### DISCUSSION TITLE
Why not encourage adding most startup apps at creation time?

### DISCUSSION THREAD

**Discussion by FlattusBlastus** [nextcloud/all-in-one#2193] [Questions] [answered]
For myself, I want a well rounded NC at time of creation. Adding these saves hours and hours of loading them one at a time. Can we promote this as a best practice?

-e NEXTCLOUD_STARTUP_APPS="twofactor_totp user_usage_report libresign quota_warning deck files_scripts calendar_resource_management calendar_news gestion files_lock memories notes files_zip calendar cloud_py_api group_default_quota notify_push timemanager files_archive camerarawpreviews files_accesscontrol richdocuments recognize tasks user_migration files_3dmodelviewer files_notify_redis groupfolders groupquota analytics dicomviewer previewgenerator workflow_script files_mindmap carnet permissions_overwrite login_notes emlviewer ransomware_protection files_linkeditor files_automatedtagging external side_menu contacts mediadc externalportal integration_youtube files_downloadactivity tvshownamer approval forms sharerenamer geoblocker integration_onedrive files_antivirus files_confidential mail collectives phonetrack files_fulltextsearch music sociallogin appointments cospend fulltextsearch_elasticsearch maps google_synchronization keeweb cadviewer tables event_update_notification ncdownloader flow_notifications files_photospheres announcementcenter fulltextsearch bookmarks unsplash files_bpm registration polls workspace files_fulltextsearch_tesseract metadata workflow_pdf_converter workflow_ocr breezedark webhooks" ^

These will also need:
-e NEXTCLOUD_ADDITIONAL_APKS="imagemagick aria2 yt-dlp ocrmypdf tesseract-ocr-data-deu sane-utils" ^

**Comment by szaimen** [comment, marked as answer]
Because you can still add the apps that you want later on using the GUI. But feel free to add all these to your installations by using this feature.
